main/main.c

```diff
@@ -747,6 +747,7 @@ PHP_INI_BEGIN()
 	STD_PHP_INI_ENTRY("input_encoding",			NULL,			PHP_INI_ALL,	OnUpdateInputEncoding,				input_encoding,		php_core_globals, core_globals)
 	STD_PHP_INI_ENTRY("output_encoding",		NULL,			PHP_INI_ALL,	OnUpdateOutputEncoding,				output_encoding,	php_core_globals, core_globals)
 	STD_PHP_INI_ENTRY("error_log",				NULL,		PHP_INI_ALL,		OnUpdateErrorLog,			error_log,				php_core_globals,	core_globals)
+	STD_PHP_INI_ENTRY("error_date_format",		"d-M-Y H:i:s e",		PHP_INI_ALL,		OnUpdateStringUnempty,			error_date_format,				php_core_globals,	core_globals)
```

Comment:
Would it make sense to have some kind of validation if the ini value is a valid date format? How is it working now if the input is not a valid format. Is the `error_time_str` below an empty string? It might be good to cover the invalid formats by test as well.

Reply:
`php_format_date` is what the PHP function `date()` also calls, so it just replaces the characters it knows to translte and otherwise keeps them:

```
$ php -r 'echo date("hello world");'
07Europe/BerlinSundaySunday2019 02019Sun, 03 Mar 2019 19:40:43 +0100Sunday03
```
Its just gibberish, but it doesn't cause any problems, I don't think validation is necessary?